common/src/main/scala/com/gu/media/pluto/PlutoProjectDataStore.scala

```diff
@@ -3,8 +3,9 @@ package com.gu.media.pluto
 import com.amazonaws.services.dynamodbv2.model.DeleteItemResult
 import com.gu.media.aws.DynamoAccess
 import com.gu.media.logging.Logging
+import com.gu.media.pluto.PlutoItem.numericIdsOnlyFilter
 import org.scanamo.{Scanamo, Table}
-import org.scanamo.DynamoFormat._
+import org.scanamo.DynamoFormat.{enumDynamoFormatCNil, _}
```

Comment:
minor: is `enumDynamoFormatCNil` needed?

Reply:
It isn't :)